apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/pac/ClaimE2E.java

```diff
@@ -130,11 +276,40 @@ public void testClaimFindByPatientWithNoResultsAndPaginationRequestedExpect200()
     given()
         .spec(requestAuth)
         .expect()
-        .log()
-        .ifError()
         .body("total", equalTo(0))
         .statusCode(200)
         .when()
         .get(requestString);
   }
+
+  /**
+   * Verifies the Claim read response returns a 200 and the json response matches the given response
+   * file's json.
+   *
+   * @param id the id to pass to the read endpoint
+   * @param includeTaxNumbers the value to use for IncludeTaxNumbers header
+   * @param expectedResponseFileName the name of the response file to compare against
+   */
+  private void verifyReadResponseMatchesFor(
+      String id, boolean includeTaxNumbers, String expectedResponseFileName) {
+
+    String requestString = claimEndpoint + id;
+
+    String response =
+        given()
+            .spec(requestAuth)
+            .header(CommonHeaders.HEADER_NAME_INCLUDE_TAX_NUMBERS, includeTaxNumbers)
+            .expect()
+            .statusCode(200)
+            .when()
+            .get(requestString)
+            .then()
+            .extract()
+            .response()
+            .asString();
+
+    String expected = rdaTestUtils.expectedResponseFor(expectedResponseFileName);
+
+    AssertUtils.assertJsonEquals(expected, response, IGNORE_PATTERNS);
+  }
```

Comment:
I like this utility method.  It makes the test far more readable.  One suggestion.  With a little refactor likeadding requestString as a parameter, adding a Set of addtional ignore patterns (possibly empty), and maybe making includeTaxNumbers nullable or optional I think it could be used in the remaining tests that don't call this method.

Reply:
Good idea; implemented it for the the search by mbi tests.